playbooks/roles/haveged/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: 'Install Haveged (faster population of PRNG)'
+  yum:
+    name: haveged
+    state: latest
```

Comment:
Thats how it was in the freeipa role.

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_